backend/tests/v5/helper/services.js

```diff
@@ -164,12 +172,22 @@ ServiceHelper.generateUserCredentials = () => ({
 	},
 });
 
-ServiceHelper.generateRevisionEntry = (isVoid = false) => ({
-	_id: ServiceHelper.generateUUIDString(),
-	tag: ServiceHelper.generateRandomString(),
-	author: ServiceHelper.generateRandomString(),
-	timestamp: ServiceHelper.generateRandomDate(),
-	void: !!isVoid,
+ServiceHelper.generateRevisionEntry = (isVoid = false, hasFile = true) => {
+	const _id = ServiceHelper.generateUUIDString();
+	return {
+		_id,
+		tag: ServiceHelper.generateRandomString(),
+		author: ServiceHelper.generateRandomString(),
+		timestamp: ServiceHelper.generateRandomDate(),
+		void: !!isVoid,
+		rFile: hasFile ? [`${_id}P2006823-BDP-XX-A-Stanmore-Envelope_optimized_ifc`] : undefined,
```

Comment:
should only has this entry if it has the file - we shouldn't be storing `rFile: undefined` anywhere.
```suggestion
		...(hasFile ? {rFile: [`${_id}P2006823-BDP-XX-A-Stanmore-Envelope_optimized_ifc`]} : {})
```

Reply:
why do we need to hardcode in `P2006823-BDP-XX-A-Stanmore-Envelope_optimized_ifc` ?